images/minio/tests/main.tf

```diff
@@ -4,11 +4,13 @@ terraform {
   }
 }
 
-variable "digest" {
-  description = "The image digest to run tests over."
+variable "digests" {
+  description = "The image digests to run tests over."
```

Comment:
generally this enumerates the keys in a type, check one of the example images.

Reply:
Done.